test_plone50.cfg

```diff
@@ -6,9 +6,9 @@ extends =
     base.cfg
 
 [versions]
-zc.buildout = 2.13.1
-setuptools = 40.8.0
+# Keep sync with requirements.txt to avoid infinite loop in buildout
+setuptools = 42.0.2
+zc.buildout = 2.13.3
 
-configparser = 4.0.2
 plone.testing = 5.0.0
-pycodestyle = 2.6
+pycodestyle =
```

Comment:
I am not sure why this is unpinned, can't we specify a proper version?

Reply:
The problem is that https://dist.plone.org/release/5.0-latest/versions.cfg pinn pycodestyle = 2.3.1 and flake8 requires the minimum version as pycodestyle>=2.7.0. It pinn a maximum version as well. As flake8 and pycodestyle are released close by, this maximum version is the last version most of the time. So I saw no problem with that. So I see no problem with that. I tried to pin flak8 but got into a hell of dependencies. So this is more practical.

Reply:
@ale-rt I did this at buildout.plonetest:

https://github.com/collective/buildout.plonetest/pull/75

Reply:
@ale-rt  as the unpin is in the `buildout.plonetest` now, I removed it from here.